requirements.txt

```diff
@@ -16,3 +16,4 @@ types-pytz
 types-requests
 gitlint>=0.13.0
 -r ./zulip/integrations/bridge_with_matrix/requirements.txt
+litellm
```

Comment:
I think we don't want to add this here; it's a huge dependency. Instead let's just make the instructions `pip install zulip litellm`.

Reply:
Added `pip install -r requirements.txt` to README.